### DISCUSSION TITLE
is impossible nuxi build --watch in nuxt3?

### DISCUSSION THREAD

**Discussion by jtk-intelights** [nuxt/nuxt#17156] [Questions]
I want to real time .vue file build in development mode.

I want to .output foler be changed when I change vue file and I press ctrl + s .

I don't want to be changed to .nuxt foler.

I want to .output foler be changed.

is it possible in nuxt3?

**Comment by TheAlexLichter** [comment]
You could try to write a script in the `package.json` with the help of `nodemon`.
Be aware that building the whole project on every change is taking way more time and has no HMR.

Example (untested):

```
{
  scripts: {
    "build:watch": "nodemon -w './*.vue' --exec \"nuxi build\""
  }
}
```

**Reply by artel1992** [reply]
Hello.
Will it be possible to run through the CLI?
